test/elixir-mode-indentation-test.el

```diff
@@ -1229,10 +1231,36 @@ defmodule Hello do
   def on?, do: true
   defmacro switch,
     do: on!
+  defmacro another,
+    do: on!
   defp self, do: value
   defmacrop whatever, do: do_it!
 end")
 
+(elixir-def-indentation-test indent-multiline-when
+                             (:tags '(indentation))
```

Comment:
You can tag the test with `:expected-result :failed` which will make it easier to merge the pull request and work on the failing test. Example: https://github.com/elixir-lang/emacs-elixir/blob/master/test/elixir-mode-indentation-test.el#L281-L301